server/vue-middleware.js

```diff
@@ -106,6 +118,7 @@ export default function createMiddleware({ config, vite }) {
 			locale: req.locale,
 			device,
 			cdnNotedLoggedIn,
+			forceHeader,
```

Comment:
In CPS we just pass the headers through and use those in the experiment utils. Is that not a good option here?